README.md

```diff
@@ -136,6 +137,31 @@ config.vm.box = "uwbbi/bionic-arm64"
 ```
 in the [Vagrantfile.vmware_ubuntu-1804](Vagrantfile.vmware_ubuntu-1804) file as you cannot use an amd64-based installation on Apple Silicon (ARM-based) hardware.
 
+## libvirt
+
+Another alternative to VirtualBox and VMware is to use the [libvirt virtualisation API](https://libvirt.org/index.html), which also has a [Vagrant plugin](https://vagrant-libvirt.github.io/vagrant-libvirt/). You will need to install libvirt on your host system. You should then edit the [Vagrantfile](Vagrantfile) to point to the [Vagrantfile.libvirt_ubuntu-1804](Vagrantfile.libvirt_ubuntu-1804) file
+```
+load 'Vagrantfile.libvirt_ubuntu-1804'
```

Comment:
Earlier in this file we recommend this approach:
```
export VAGRANT_VAGRANTFILE=Vagrantfile.libvirt_ubuntu-1804
```
Do you not like that method?
It's just a bit strange recommending different approaches in different sections.

Not a big deal.